test/clj/web/user/test.clj

```diff
@@ -0,0 +1,16 @@
+(ns web.user.test
+  (:require 
+   [web.user :refer :all]
+   [clojure.test :refer :all]))
+
+(deftest username
+  (testing "valid-username?"
+    (is (valid-username? "test"))
+    (is (not (valid-username? "abcdefghijklmnopqrstu")) "Username is invalid when more than 20 characters")
+    (is (valid-username? "abcdefghijklmnopqrst") "Username is valid when less than or equal to 20 characters")
+    (is (valid-username? "🐸") "Emoji are valid in usernames")
+    (is (valid-username? "🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸") "Make sure 20 glyphs is still valid")
+    (is (valid-username? "<script src=\"t.js\"") "HTML-like text is valid")
+    (is (valid-username? "<h1>hello</h1>") "HTML is not valid")
+    (is (valid-username? "http://example.org") "URLs are not valid")))
```

Comment:
This should fall so use not

Reply:
Blah silly me. Fixed it and found out I needed `re-find` instead of `re-matches` - all passing now :slightly_smiling_face: